packages/cli/src/commands/help.ts

```diff
@@ -277,9 +282,25 @@ export function buildCommandExampleLines(command: Command) {
   return outputArrayToString(outputArray);
 }
 
-function buildDescriptionLine(command: Command) {
-  const line: string[] = [INDENT, command.description, NEWLINE];
-  return lineToString(line);
+function buildDescriptionLine(
+  command: Command,
+  options: BuildHelpOutputOptions
+) {
+  const _tableOptions = Object.assign({}, tableOptions, {
+    colWidths: [options.columns],
+    style: {
+      'padding-left': INDENT.length,
+    },
+  });
+  const table = new Table(_tableOptions);
+  table.push([
+    {
+      content: command.description,
+      wordWrap: true,
+    } as _CellOptions,
+  ]);
```

Comment:
**suggestion (soft-blocking):** extract function

Can we extract this into a named function to make it super clear that we're using this table module purely for word wrapping?‏

Reply:
**alternate suggestion (non-blocking):** use `word-wrap`

I don't think the module [word-wrap](https://github.com/jonschlinkert/word-wrap) is [too big](https://packagephobia.com/result?p=word-wrap) to bring in. I'd be fine with that.

Reply:
If we use `word-wrap` will the wrapped description continue to be indented on subsequent lines?

Reply:
@cb1kenobi yessir.

Reply:
Tested out `word-wrap` and hilariously it does a poorer job unless you manually add a bit of right padding.
<img width="544" alt="CleanShot 2023-08-18 at 13 34 56@2x" src="https://github.com/vercel/vercel/assets/9736/622a0033-e5ff-43ba-873d-fc6120fd3eec">

I do like Sean's suggestion of extracting into a function.